src/input.jsx

```diff
@@ -121,7 +121,7 @@ class Input extends React.Component {
 
     return <input className={classes}
       ref={i => this.input = i}
-      type='text'
+      type={this.props.inputType}
```

Comment:
Is there a reason to make this a prop? Can we change this for all without breaking anything?